File: WaterManagerUI/Model/Security/LoginRequestDTO.cs
using System.ComponentModel.DataAnnotations;

namespace WaterManagerUI.Model.Security;

public class LoginRequestDTO
{
    [Required] public String username { get; set; }
    [Required] public String password { get; set; }

    public LoginRequestDTO(string username, string password)
    {
        this.username = username;
        this.password = password;
    }

    public LoginRequestDTO()
    {
    }
}

File: WaterManager/src/main/java/pissir/watermanager/security/model/LoginRequestDTO.java
package pissir.watermanager.security.model;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * @author Almasio Luca
 * @author Borova Dritan
 * @author Gattico Alessandro
 */

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class LoginRequestDTO {
	
	private String username;
	private String password;
	
}
